projects/Mallard/src/screens/settings/dev-zone.tsx

```diff
@@ -125,6 +125,15 @@ const DevZone = withNavigation(({ navigation }: NavigationInjectedProps) => {
                 >
                     Storybook
                 </Button>
+                <Button
+                    onPress={() => {
+                        navigation.navigate(
+                            routeNames.onboarding.OnboardingConsent,
+                        )
+                    }}
+                >
+                    Show Startup Consent
+                </Button>
```

Comment:
Another existing alternative to this is to press `Reset` at the bottom of the privacy setting page and reload the app.

Reply:
True - but not if you are on the internal beta. Only shows up on the simulator.